astropy/utils/xml/writer.py

```diff
@@ -148,6 +153,20 @@ def start(self, tag, attrib={}, **extra):
         return len(self._tags)
 
     @contextlib.contextmanager
+    def xml_escaping(self, enabled=True, clean_kwargs=None):
+        current_xml_escape_cdata = self.xml_escape_cdata
+        if not enabled:
+            if HAS_BLEACH:
+                if clean_kwargs is None:
+                    clean_kwargs = {}
+                self.xml_escape_cdata = lambda x: bleach.clean(x, **clean_kwargs)
+            else:
+                raise ValueError('bleach package is required when HTML escaping is disabled.\n'
+                                 'Use "pip install bleach".')
```

Comment:
Or `def select_xml_cleaning_method(method='all', **kwargs)`.  Then `method` could be `whitelist` in which case it would `bleach`, or `method='all'` wherein it uses the default brute-force way.